### ISSUE TITLE
Switch language on the Build

### ISSUE THREAD

**Issue by ZeTheGreat** [kamilglod/i18next-loader#1]
Hello!!!! 

I want to use this i18next application, but to do it, I need to do this little change:

```
module.exports = function(source) {
  if (this.cacheable) {
    this.cacheable();
  }

  var params  = loaderUtils.parseQuery(this.query);
  var funcName = params.funcName || '__';
  var defaultQuotes = params.quotes || '\'';
  var quotes = params.quotes || '\'';
  var regex = new RegExp(funcName + '\\(([\'"`])(.*?)[\'"`](?:,\\s(.*?))?\\)', ['g']);

  params.lng && i18next.changeLanguage(params.lng) // Add this validation to choose which language I want in the compilation
  
  source = source.replace(regex, function (res, quotes, key, jsonString) {

    if (jsonString) {
      var options = {};
      jsonString.replace(jsonRegex, function (res, key, value) {
        options[key] = value;
      });
    } else {
      var options = undefined;
    }

    quotes = (quotes == '`') ? '`' : defaultQuotes;
    return quotes + i18next.t(key, options) + quotes;
  });

  return source;
}
```

**Comment by ZeTheGreat** [comment]
PR: https://github.com/kamilglod/i18next-loader/pull/2

**Comment by ZeTheGreat** [comment]
DOne

**Comment by kamilglod** [comment]
Just released the 0.1.0 version, please verify if it's installable and it's working. I don't even remember what was the release process so I just run `npm publish`

**Comment by ZeTheGreat** [comment]
Tested it is working :)
